README.md

```diff
@@ -4,28 +4,33 @@ https://github.com/pingcap/chaos-mesh
 ## Install Gafana Operator
 
 ```shell
-oc new-project sre-monitoring
-oc apply -f grafana-operator.yaml -n sre-monitoring
-#fixes some operator's bug
-oc scale deployment grafana-operator --replicas=0
-#oc set image deployment/grafana-operator grafana-operator=quay.io/integreatly/grafana-operator:v3.0.0-beta
-oc set image deployment/grafana-operator grafana-operator=quay.io/integreatly/grafana-operator:v1.4.0
-oc set image deployment/grafana-operator grafana-operator=quay.io/integreatly/grafana-operator:v3.2.0
-oc scale deployment grafana-operator --replicas=1
+git clone git@github.com:integr8ly/grafana-operator.git
+
+cd grafana-operator/
+
+ansible-playbook deploy/ansible/grafana-operator-cluster-resources.yaml \
```

Comment:
can we switch to the non ansible base install.
also please remove references to a particular cluster.

Reply:
added Manual and Automated instructions